sycl/include/sycl/ext/oneapi/experimental/fixed_size_group.hpp

```diff
@@ -137,57 +137,57 @@ template <size_t PartitionSize, typename ParentGroup> class fixed_size_group {
   fixed_size_group() {}
```

Comment:
This constructor should probably be renamed

Reply:
didn't plan to change the name of file. but yes. it should be changed along with the class name including all refs.

I hope it's done now